src/data/colleges/en.ts

```diff
@@ -10,22 +10,40 @@ import {
 import { AdvisorGroup } from '../../tools/advisors/types';
 import { lastNameRange, lastNameRanges } from '../../tools/advisors/checkers';
 
-const engineeringLiberalArtsDistributions: readonly string[] = [
-  'CA',
-  'HA',
-  'LA',
-  'LAD',
-  'KCM',
-  'SBA',
-  'FL',
-  'CE',
-  'ALC',
-  'SCD',
-  'HST',
-  'ETM',
-  'SSC',
-  'GLC',
-];
+const engineeringLiberalArtsGroups: Record<string, string[]> = {
+  'Group 1': [
+    'CA-AAP',
```

Comment:
Hmmm I'm not sure if it is strictly necessary to go and add in the content after the hyphen for all of these categories. `hasCategory` checks for a substring match, and all the `CA`s are in `Group 1`, all the `HA`s in `Group 2` ... (they are disjoint sets) so such granularity might not be needed. @nidhi-mylavarapu not sure if you have any thoughts?

Reply:
I tried the groups without the after-hyphen content and it didn't filter correctly...not sure if it's something else though?

Reply:
Alright we can probably stick with just this then, I don't anticipate the categories changing much.